src/OpenConext/ProfileBundle/Resources/views/layout.html.twig

```diff
@@ -91,7 +85,5 @@
 {% endblock %}
 
 {% block javascripts %}
-    <script src="{{ asset('js/material.min.js') }}"></script>
-    <script src="{{ asset('js/jquery-3.4.1.min.js') }}"></script>
-    <script src="{{ asset('js/main.js') }}"></script>
+    <script src="{{ asset('build/app.js') }}"></script>
```

Comment:
Encore has some nice helper functions for the includes, it may be better to use those